src/CollapseUIButton.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useRef } from 'react'
```

Comment:
I don't see where this component `CollapseUIButton` is created.

Reply:
Same. I worked on it just in case.